app/views/_container-statuses.html

```diff
@@ -0,0 +1,48 @@
+<!--
```

Comment:
Caught ```podStatus``` earlier, should be updated now... if we're going for only listing values/properties/variables used in the templates can adjust accordingly